AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -0,0 +1,142 @@
+    
+    var enginePickerView = UIPickerView()
+    var languagePickerView = UIPickerView()
+    var regionPickerView = UIPickerView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        meetingId = self.model!.meetingId
+        engineTextField.inputView = enginePickerView
+        languageTextField.inputView = languagePickerView
+        regionTextField.inputView = regionPickerView
+        
+//        engineTextField.placeholder = "Select Engine"
+//        languageTextField.placeholder = "Select Language"
+//        regionTextField.placeholder = "Select Region"
+        
+        engineSelected = "transcribe"
+        languageSelected = "en-US"
+        regionSelected = "us-east-1"
+        
+        engineTextField.text = "transcribe"
+        languageTextField.text = "en-US"
+        regionTextField.text = "us-east-1"
+        
+        engineTextField.textAlignment = .center
+        languageTextField.textAlignment = .center
+        regionTextField.textAlignment = .center
+        
+        enginePickerView.delegate = self
+        languagePickerView.delegate = self
+        regionPickerView.delegate = self
+        enginePickerView.dataSource = self
+        languagePickerView.dataSource = self
+        regionPickerView.dataSource = self
+        
+        enginePickerView.tag = 1
+        languagePickerView.tag = 2
+        regionPickerView.tag = 3
+    }
+    @IBAction func startTranscriptionButton(_ sender: UIButton) {
+        var url = AppConfiguration.url
+        url = url.hasSuffix("/") ? url : "\(url)/"
+        let encodedURL = HttpUtils.encodeStrForURL(
+            str: "\(url)start_transcription?title=\(meetingId)&language=\(languageSelected)&region=\(regionSelected)&engine=\(engineSelected)")
+        HttpUtils.postRequest(url: encodedURL, jsonData: nil) {_,_ in
+
+            }
+        MeetingModule.shared().dismissTranscription(self)
+    }
+    
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+    }
+    
+    override  func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+}
+
+extension LiveTranscriptionOptionsViewController: UIPickerViewDataSource, UIPickerViewDelegate {
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        switch pickerView.tag {
+        case 1:
+            return engines.count
+        case 2:
+            return languages.count
+        case 3:
+            return regions.count
+        default:
+            return 1
+        }
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        switch pickerView.tag {
+        case 1:
+            return engines[row]
+        case 2:
+            return languages[row]
+        case 3:
+            return regions[row]
```

Comment:
Will this cause any problem if we switch from Transcribe to Transcribe Medical in "engine" since they have different length of languages and regions?